uaclient/util.py

```diff
@@ -208,17 +210,21 @@ def readurl(url: str, data: 'Optional[bytes]' = None,
     return content, resp.headers
 
 
-def _subp(args: 'Sequence[str]', rcs: 'Optional[List[int]]' = None,
-          capture: bool = False) -> 'Tuple[str, str]':
+def _subp(args: 'Sequence[str]',
+          rcs: 'Optional[List[int]]' = None,
+          capture: bool = False,
+          timeout: 'Optional[int]' = None) -> 'Tuple[str, str]':
```

Comment:
Definitely not a big deal, but timeout on communicate is actually a float; no need for us to be more restrictive: https://github.com/python/typeshed/blob/9941616b9efd31181bf3e5dd1d5f375791352fec/stdlib/3/subprocess.pyi#L318-L322